utils.go

```diff
@@ -55,11 +56,16 @@ func CreateResourceLabels(resourceID string) map[string]string {
 func CreateAllResourceLabelsFrom(rm resourceMeta) map[string]string {
 	formatTag := "pretty"
 	labels := make(map[string]string)
-	split := strings.Split(rm.ResourceURL, "/")
+	split := strings.Split(rm.resourceURL, "/")
 	labels["resource_group"] = split[resourceGroupPosition]
 
-	for k, v := range rm.Resource.Tags {
+	for k, v := range rm.resource.Tags {
 		k = strings.ToLower(k)
+
+		if !invalidLabelPrefix.MatchString(k) {
+			k = "_" + k
```

Comment:
invalidLabelPrefix contains _ and then we add one after ? not sure I understand then should'nt the regex be named validPrefix ?